### PR TITLE
Trying to make a pull request

### PR DESCRIPTION
**Pull request by nitinprakash96**

### CODE REVIEW

File: PULLREQUEST.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# zap-pull-request
```
Comment (nitinprakash96) [inline PULLREQUEST.md:1]: Add some contents to it

Comment (jerry-137) [inline PULLREQUEST.md:1]: Yes! There's defenitely some conent missing here!

**Review by jerry-137 — CHANGES_REQUESTED** [review]

File: PULLREQUEST.md [hunk 2]
```diff
@@ -1 +1,2 @@
 # zap-pull-request
+## zap-pull-request
```
Comment (jerry-137) [inline PULLREQUEST.md:2]: Why the same change @nitinprakash96

**Review by jerry-137 — CHANGES_REQUESTED** [review]

File: PULLREQUEST.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# zap-pull-request
+## zap-pull-request
```
Comment (jerry-137) [inline PULLREQUEST.md:2]: Wrap it in 72 chars!

**Review by jerry-137 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**